spec/app/tournament/tournament.spec.js

```diff
@@ -0,0 +1,121 @@
+import {Tournament} from "../../../app/tournament/tournament";
+import {Player} from "../../../app/tournament/player";
+
+describe('class Tournament', function() {
+
+    var spiderman = new Player("Spiderman", 1);
+    var batman = new Player("Batman", 3);
+    var theThing = new Player("The Thing", 5);
+
+    var tournament;
+
+    beforeEach( function() {
+        tournament = new Tournament();
+    });
+
+    function setMockedTournament(players) {
+        while (players--) {
+            tournament.addPlayer(spiderman);
+        }
+    }
+
+    it('should return false when searching for a non-enrolled player', function() {
+        expect( tournament.existsPlayer(spiderman) ).toBeUndefined();
```

Comment:
No, I'm fine returning undefined/Player. But that's not the spec, that's the implementation. Assert what you expect, not what you do. You should be able to change from false/true to undefined/object without breaking the spec.